modules/functional-test/src/main/java/com/intuit/wasabi/tests/service/IntegrationMutualExclusion.java

```diff
@@ -48,378 +51,443 @@
-        Assert.assertEquals(response.jsonPath().getString("exclusions[0].status"), "FAILED", "Status should be FAILED.");
-        Assert.assertTrue(response.jsonPath().getString("exclusions[0].reason").contains("is in TERMINATED or DELETED state"), "Reason not valid.");
-
-        // check DELETED experiment
-        deleteExperiment(batchMutExExperiments.get(0));
-        postExclusions(batchMutExExperiments.get(0), Collections.singletonList(batchMutExExperiments.get(1)), HttpStatus.SC_NOT_FOUND);
-        postExclusions(batchMutExExperiments.get(1), Collections.singletonList(batchMutExExperiments.get(0)), HttpStatus.SC_CREATED);
-        Assert.assertEquals(response.jsonPath().getString("exclusions[0].status"), "FAILED", "Status should be FAILED.");
-        Assert.assertTrue(response.jsonPath().getString("exclusions[0].reason").contains(" not found."), "Reason not valid.");
-
-        // invalid ID
-        Experiment invalidExperiment = ExperimentFactory.createExperiment().setId("7b471e3a-fae7-428c-90d1-02c3f6a0b441");
-        // check invalid ID
-        postExclusions(invalidExperiment, batchMutExExperiments.subList(1, batchMutExExperiments.size()), HttpStatus.SC_NOT_FOUND);
-
-        // check deletion with invalid ID left and right
-        deleteExclusion(batchMutExExperiments.get(1), invalidExperiment, HttpStatus.SC_NOT_FOUND);
-        deleteExclusion(invalidExperiment, batchMutExExperiments.get(1), HttpStatus.SC_NOT_FOUND);
-
-        // check GET for invalid ID
-        getExperiment(invalidExperiment, HttpStatus.SC_NOT_FOUND);
-
-        // passed End time
-        Experiment endTimePassedExperiment = postExperiment(
-                ExperimentFactory.createExperiment()
-                    .setEndTime(TestUtils.relativeTimeString(-3))
-                    .setStartTime(TestUtils.relativeTimeString(-4)));
-        toCleanUp.add(endTimePassedExperiment);
-        postBuckets(BucketFactory.createBuckets(endTimePassedExperiment, 2));
-        putExperiment(endTimePassedExperiment.setState(Constants.EXPERIMENT_STATE_RUNNING));
-        postExclusions(endTimePassedExperiment, Collections.singletonList(experimentDefApp), HttpStatus.SC_NOT_FOUND);
-
-        // different applications
-        postExclusions(experimentDefApp, Collections.singletonList(experimentOtherApp), HttpStatus.SC_CREATED);
-        Assert.assertEquals(response.jsonPath().getString("exclusions[0].status"), "FAILED", "Status should be FAILED.");
-        Assert.assertTrue(response.jsonPath().getString("exclusions[0].reason").contains("Mutual exclusion rules can only be defined for experiments within the same application."), "Reason not valid.");
-
-    }
-
-
-    /**
-     * Cleans up.
-     */
-    @AfterClass
-    public void cleanup() {
-        toCleanUp.addAll(batchMutExExperiments);
-    }
+	private static final Logger LOGGER = getLogger(IntegrationMutualExclusion.class);
```

Comment:
Do you mind turning in you IDE the tabs to spaces? You can then autoformat the file again push it (maybe when the correct response code is clear).